operations/node_management/Swap_a_Compute_Blade_with_a_Different_System_Using_SAT.md

```diff
@@ -0,0 +1,472 @@
+
+   ```bash
+   ncn# sat swap blade -a disable x9000c3s0
+   ```
+
+   This command will save the MAC addresses, IP addresses, and node component names (xnames) from the blade to a JSON document. The document is stored in a file with the following naming convention:
+
+   ```text
+   ethernet-interface-mappings-<blade_xname>-<current_year>-<current_month>-<current_day>.json
+   ```
+
+   If a mapping file already exists with the above name, then a numeric suffix will be appended to the file name in front of the `.json` extension.
+
+   The mapping file should be copied to the NCN on the destination system used for the swap procedure, if necessary. In this example, the
+   filename is changed to `ethernet-interface-mappings-src.json` on the destination system for clarity.
+
+   ```bash
+   ncn# scp ethernet-interface-mappings-x3000c3s0-2022-01-01.json <dest>-ncn-m001:ethernet-interface-mappings-src.json
+   ```
+
+### Source: Remove the blade
+
+1. Remove the blade from the source system.
+   Review the *Remove a Compute Blade Using the Lift* procedure in *HPE Cray EX Hardware Replacement Procedures H-6173* for detailed instructions for replacing liquid-cooled blades ([HPE Support](https://internal.support.hpe.com/)).
+1. Drain the coolant from the blade and fill with fresh coolant to minimize cross-contamination of cooling systems.
+   Review *HPE Cray EX Coolant Service Procedures H-6199*. If using the hand pump, review procedures in the *HPE Cray EX Hand Pump User Guide H-6200* ([HPE Support](https://internal.support.hpe.com/)).
+1. Install the blade from the source system in a storage rack or leave it on the cart.
+
+## Remove blade from destination system
+
+### Destination: Prepare blade for removal
+
+1. Using the work load manager (WLM), drain running jobs from the affected nodes on the blade.
+
+   Refer to the vendor documentation for the WLM for more information.
+
+1. Shut down the nodes on the target blade.
+
+   Use the `sat bootsys` command to shut down the nodes on the target blade. Specify the appropriate component
+   name (xname) for the slot and a comma-separated list of the appropriate BOS session templates for the nodes on the blade.
+
+   The appropriate BOS session templates should be determined using the same procedure as was used to [determine the appropriate BOS session templates on the source system](#determine-bos-session-template).
+
+   ```bash
+   ncn# BOS_TEMPLATES=cos-2.0.30-slurm-healthy-compute
+   ncn# sat bootsys shutdown --stage bos-operations --bos-limit x1005c0s3 --recursive --bos-templates $BOS_TEMPLATES
+   ```
+
+### Destination: Use SAT to remove the blade from hardware management
+
+1. Power off the slot and delete blade information from HSM.
+
+   Use the `sat swap` command to power off the slot and delete the blade's Ethernet interfaces and Redfish endpoints from HSM.
+
+   ```bash
```

Comment:
I know you're just following convention here but I really don't care for using `bash` to highlight console commands. For this command you get the word 'enable' randomly highlighted because it's a bash builtin despite having no special significance in this context. `bash` should be for bash scripts imo.

Reply:
Yeah, I don't disagree. As you note here I am mostly trying to conform with the existing usage and conventions.